release/RELEASE.md

```diff
@@ -35,8 +35,9 @@ patches we backport to the 1.0 release branch).
    label.
 
 1. Once the PR has built, check that it was considered a release build by our
-   CI. You can do that by looking at the output of the `check_for_release`
-   build step.
+   CI. If you are working from an automated PR, check that it sent a message to
+   \#team-daml to say it has finished building. If the PR was manually created,
```

Comment:
```suggestion
   `#team-daml` to say it has finished building. If the PR was manually created,
```

Reply:
Done, and updated all other Slack channel references to match.